package.json

```diff
@@ -58,13 +68,15 @@
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
     "test": "grunt",
-    "coverage": "grunt coverage"
+    "coverage": "grunt coverage",
+    "test:cli": "grunt build && npm link && qunit test/cli/*.js"
```

Comment:
It failed on a previous run. We can mess around with it after this lands, but for now I don't feel it is worth digging into.